ansible/roles/helios_broker_setup/defaults/main.yml

```diff
@@ -0,0 +1,81 @@
+
+dockerhub_user_name: changeme
+dockerhub_user_password: changeme
+helios_dockerhub_org: projecthelios
+
+asb_src_dir: /tmp/aws-service-broker
+
+helios_broker_image_name: "docker.io/{{ helios_dockerhub_org }}/aws-service-broker"
+helios_broker_tag: "latest"
+helios_broker_image: "{{ helios_broker_image_name }}:{{ helios_broker_tag }}"
+
+#
+# Template defaults
+#
+
+# Path of the etcd binary
+etcd_path: /usr/local/bin/etcd
+
+# Configuration filepath for Ansible Service Broker
+broker_config: /etc/ansible-service-broker/config.yaml
+
+# SSL enabled: insecure="false", termination=Reencrypt
+# Insecure enabled: insecure="true", termination=edge
+# Run in insecure mode
+insecure: "false"
+termination: Reencrypt
+
+# APB Image Tag
+tag: latest
```

Comment:
Unsure what was using 'tag', was this supposed to be 'apbtag', 
If so, assume we should call it "helios_apbtag"

Reply:
will update